### ISSUE TITLE
 About ground truth label

### ISSUE THREAD

**Issue by HHeracles** [cassie1728/ESIR-a-little-impove#3]
How is the ground truth label used in training generated? In addition, what is the loss function？
